smart-accounts-kit/guides/advanced-permissions/execute-on-metamask-users-behalf.md

```diff
@@ -236,6 +238,7 @@ To redeem the permissions, use the client action based on your session account t
 A smart account uses the Bundler Client's [`sendUserOperationWithDelegation`](../../reference/advanced-permissions/bundler-client.md#senduseroperationwithdelegation) action,
 and an EOA uses the Wallet Client's [`sendTransactionWithDelegation`](../../reference/advanced-permissions/wallet-client.md#sendtransactionwithdelegation) action:
 
+See [`sendUserOperationWithDelegation`](../../reference/advanced-permissions/bundler-client.md#senduseroperationwithdelegation) and [`sendTransactionWithDelegation`](../../reference/advanced-permissions/wallet-client.md#sendtransactionwithdelegation) API reference to learn more.
```

Comment:
ditto comment above: the more verbose style not needed, but consider extending the linkable text to include "Bundler Client's" and "Wallet Clients" if the motivation is to make link existence clearer.